Tests/SourceGenerators/SourceGeneratorAssemblyToProcess/Generated/Realm.SourceGenerator/Realms.SourceGenerator.RealmGenerator/NamespaceObj_generated.cs

```diff
@@ -0,0 +1,289 @@
+﻿// <auto-generated />
+using SourceGeneratorAssemblyToProcess;
+using SourceGeneratorAssemblyToProcess.Generated;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+using System.Xml.Serialization;
+using System.Reflection;
+using System.ComponentModel;
+using Realms;
+using Realms.Weaving;
+using Realms.Schema;
+using OtherNamespace;
+
+namespace SourceGeneratorAssemblyToProcess
+{
+    [Generated]
+    [Woven(typeof(NamespaceObjObjectHelper))]
+    public partial class NamespaceObj : IRealmObject, INotifyPropertyChanged, IReflectableType
+    {
+        public static ObjectSchema RealmSchema = new ObjectSchema.Builder("NamespaceObj", ObjectSchema.ObjectType.RealmObject)
+        {
+            Property.Primitive("Id", RealmValueType.Int, isPrimaryKey: false, isIndexed: false, isNullable: false, managedName: "Id"),
+            Property.Object("OtherNamespaceObj", "OtherNamespaceObj", managedName: "OtherNamespaceObj"),
+        }.Build();
+
+        #region IRealmObject implementation
+
+        private INamespaceObjAccessor _accessor;
+
+        IRealmAccessor IRealmObjectBase.Accessor => Accessor;
+
+        internal INamespaceObjAccessor Accessor => _accessor = _accessor ?? new NamespaceObjUnmanagedAccessor(typeof(NamespaceObj));
+
+        [IgnoreDataMember, XmlIgnore]
+        public bool IsManaged => Accessor.IsManaged;
+
+        [IgnoreDataMember, XmlIgnore]
+        public bool IsValid => Accessor.IsValid;
+
+        [IgnoreDataMember, XmlIgnore]
+        public bool IsFrozen => Accessor.IsFrozen;
+
+        [IgnoreDataMember, XmlIgnore]
+        public Realm Realm => Accessor.Realm;
+
+        [IgnoreDataMember, XmlIgnore]
+        public ObjectSchema ObjectSchema => Accessor.ObjectSchema;
+
+        [IgnoreDataMember, XmlIgnore]
+        public DynamicObjectApi DynamicApi => Accessor.DynamicApi;
+
+        [IgnoreDataMember, XmlIgnore]
+        public int BacklinksCount => Accessor.BacklinksCount;
+
+        public void SetManagedAccessor(IRealmAccessor managedAccessor, IRealmObjectHelper helper = null, bool update = false, bool skipDefaults = false)
+        {
+            var newAccessor = (INamespaceObjAccessor)managedAccessor;
+            var oldAccessor = _accessor as INamespaceObjAccessor;
+            _accessor = newAccessor;
+
+            if (helper != null)
+            {
+
+                if(!skipDefaults || oldAccessor.Id != default(int))
```

Comment:
Can we instead generate ` ... !=  default`?

Reply:
`default` literal has been introduced in C# 7.1.  Is it old enough that we expect all the users of realm to use an older version?